packages/pangea-node-sdk/src/types.ts

```diff
@@ -676,6 +676,14 @@ export namespace Intel {
       export interface BreachedOptions extends Intel.Options {
         start?: string;
         end?: string;
+
+        /** A token given in the raw response from SpyCloud. Post this back to paginate results */
+        cursor?: string;
+      }
+
+      export interface BreachedBulkOptions extends Intel.Options {
+        start?: string;
+        end?: string;
       }
```

Comment:
This could also be:
```suggestion
      export type BreachedBulkOptions = Omit<BreachedOptions, "cursor">;
```
if you think that's cleaner, but up to you.

Reply:
Yes, I think original option is clearer. Nice suggestion anyway, thanks!